lib/rails_response_dumper/runner.rb

```diff
@@ -13,10 +13,11 @@ def initialize(options)
 
     def run_dumps
       dumps_dir = Rails.root.join('dumps')
+      file_path = options[:filename] || 'dumpers/**/*.rb'
       FileUtils.rm_rf dumps_dir
       FileUtils.mkdir_p dumps_dir
 
-      Dir[Rails.root.join('dumpers/**/*.rb')].each { |f| require f }
+      Dir[Rails.root.join(file_path)].each { |f| require f }
```

Comment:
Hmm, this doesn't filter by the passed in filename. If a dumper has a `require` on another dumper, that 2nd dumper will still run. This just limits what will be required, not what will run.

Reply:
Hi @jdufresne , just coming back to this. Was wondering if you could give an example of a case where one might require one dumper from another dumper's file and we expect it to not run?

I keep thinking about this like rspec shared examples, where I assume we'd want to run the shared tests required in the specified file.